CONTRIBUTING.md

```diff
@@ -131,14 +131,14 @@ run each time you commit changes.
 
 Compiler requirements:
 
-* `gcc`     version 5.4+
-* `nvcc`    version 10.0+
-* `cmake`   version 3.14.0+
+* `gcc`     version 9.0+
+* `nvcc`    version 11.0+
```

Comment:
Once we switch to enhanced compat, we'll likely always require the latest nvcc version. I don't know how we want to reflect that.

Reply:
For `cudf` that should be easy right? The README documents requirements for users of a pre-built version, while `CONTRIBUTING` documents requirements for building from source.

So here we could say `nvcc      latest 11.X version`

Reply:
It would be nice to build some clean warning messaging into our cmake as well instead of just possible esoteric compilation failures.